conf/modules.config

```diff
@@ -988,7 +988,7 @@ process{
     // VCF QC
     withName: 'BCFTOOLS_STATS'{
         ext.when         = { !(params.skip_tools && params.skip_tools.contains('bcftools')) }
-        ext.prefix       = { meta.type ? "${meta.variantcaller}_${vcf.baseName.minus(".vcf")}_${meta.type}" : "${meta.variantcaller}_${vcf.baseName.minus(".vcf")}" }
+        ext.prefix       = { "${vcf.baseName.minus(".vcf")}" }
```

Comment:
then why minus(".vcf") if basename here?

Reply:
here baseName removes .gz and then vcf is still left

Using `simpleName` would remove also anything after a dot in the regular filename

Reply:
will comment this better for future us